notebooks/official/workbench/gaming_churn_prediction/churn_prediction_for_game_developers.ipynb

```diff
@@ -659,7 +567,7 @@
         "id": "P358Jo_s8WC0"
       },
       "source": [
-        "You cannot simply use raw event data to train a machine learning model as it would not be in the right shape and format to use as training data. So in this section, you learn how to pre-process the raw data into an appropriate format to use as training data for classification models.\n"
+        "You can't simply use raw event data to train a machine learning model as it won't be in the right shape and format to use as training data. So in this section, you learn how to pre-process the raw data into an appropriate format to use as training data for classification models."
```

Comment:
change "as" to "since"
- "a machine learning model since it won't be in the right shape"